webapp/channels/src/components/profile_popover/profile_popover.tsx

```diff
@@ -667,11 +667,10 @@ class ProfilePopover extends React.PureComponent<ProfilePopoverProps, ProfilePop
                     key='user-popover-settings'
                     className='popover__row first'
                 >
-                    <FormattedMessage
-                        id='user_profile.account.post_was_created'
-                        defaultMessage='This post was created by an integration from'
-                    />
-                    {` @${this.props.user.username}`}
+                    {formatMessage({
+                        id: 'user_profile.account.post_was_created',
+                        defaultMessage: 'This post was created by an integration from',
+                    }).concat(` @${this.props.user.username}`)}
```

Comment:
There is no absolutely need to change this because this works correctly, but I am prevy of encouraging this pattern over existing one such as with a variable
```js
<FormattedMessage
  id="app.greeting"
  defaultMessage="Hello, {name}!"
  values={{
    name: 'Ahmed',
  }}
/>
```

Reply:
Ohh, I didnt know you can do this. I will update as suggested